projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -58,15 +58,16 @@ export const HomeScreen = ({
     const issueList = useMemo(
         () =>
             demoIssues.map(issue => ({
-                key: issue.date.toString(),
+                key: issue.key,
                 title: renderIssueDate(issue.date).date,
                 explainer: issue.key,
                 data: {
                     issue: issue.key,
                 },
             })),
-        [demoIssues.map(({ date }) => date)],
+        demoIssues.map(({ date }) => date),
```

Comment:
```suggestion
        demoIssues.map(({ key }) => key),
```

unrelated to this PR but key might be better here as it'll be unique per issue too!

Reply:
I've changed it anyway 🆗